src/site/content/en/blog/bfcache/index.md

```diff
@@ -302,27 +297,66 @@ onAllChangesSaved(() => {
 {% endCompareCaption %}
 {% endCompare %}
 
+### Minimize use of `Cache-Control: no-store`
 
-### Avoid window.opener references
+`Cache-Control: no-store` is an HTTP header web servers can set on responses that instructs the browser not to store the response in any HTTP cache. This should be used for resources containing sensitive user information, for example pages behind a login.
+
+Bfcache is not a HTTP cache but, historically, when `Cache-Control: no-store` is set on the page itself (as opposed to any subresource), browsers have not included the page in the bfcache. There is [work currently underway to change this behavior for Chrome](https://github.com/fergald/explainer-bfcache-ccns/blob/main/README.md) in a privacy-preserving manner, but at present any pages using `Cache-Control: no-store` will not get the performance benefits of the bfcache.
+
+For best performance, it is recommended that pages that do not contain sensitive user information are cacheable, even if only for a short time. This will improve performance in general, but also avoid your site becoming ineligible for the bfcache and therefore not benefiting from those instant page restores.
+
+For sites that do not contain sensitive user information but still wish to ensure stale pages are not served, using `Cache-Control: no-cache` or `Cache-Control: max-age=0` prevents the page from being used without being revalidated first. However, unlike `Cache-Control: no-store`, these directives do not prevent pages from being included in—and getting the performance benefits of—the bfcache. For back/forward navigations a page refresh may not be expected nor desirable for the user.
+
+In the next section we show how you can ensure stale data is not served if you do want to update the page on bfcache restores but still retain the performance benefits of using the bfcache.
```

Comment:
```suggestion
Note that when a page is restored from bfcache, it is restored from memory, not from the HTTP cache. As a result, directives like `Cache-Control: no-cache` or `Cache-Control: max-age=0` are not taken into account, and no revalidation occurs before the content is displayed to the user.

This is still likely a better user experience, however, as bfcache restores are instant and—since pages do not stay in the bfcache for very long—it's unlikely that the content is out of date. However, if your content does change minute-by-minute, you can fetch any updates using the `pageshow` event, as outlined in the next section.
```

Add some content explaining that cache control directives do not apply to bfcache restores (which I think is important to clarify).

Also, note that our style guide discourages the use of "we", see: https://developers.google.com/style/person